CHANGE-NOTES.md

```diff
@@ -5,6 +5,7 @@
 - Added: extra backup keyboard shortcut (`Ctrl + Option + Shift + M`) for opening Git Machete tab on macOS,
   in case `Command + Option + Shift + M` is already taken by Kotlin plugin.
 - Improved: push action is no longer blocked by the other machete actions; it can be run in parallel.
+- Added: `Yes + Quit` button in Traverse dialogs.
```

Comment:
while we are here... does it make sense to group added, improved, fixed... ? Which means to simply sort them. I think I saw that grouping in some change notes - it's easier to review them esp if you are looking for smth. probably makes it's more important when there is a lot of changes... 🤔

Reply:
Right... makes sense, lemme reorder